modules/ROOT/pages/audit-logging.adoc

```diff
@@ -0,0 +1,63 @@
+= Audit Logging
+:page-edition: Enterprise
+:description: Audit Logging provides tools for administrators to track operational irregularities and to support regulatory and security compliance standards, such as link:https://www.hhs.gov/hipaa/index.html[HIPAA] and link:https://soc2.co.uk/soc2[SOC-2].
+
+[abstract]
+{description}
+
+== About Audit Logging
+
+Key concepts, use cases and sign posting
+Explain audit log files, only one active at a time that they’re written etc
```

Comment:
These look like notes to self. Comment out so there's no risk of them being shown to users

```suggestion
// Key concepts, use cases and sign posting
// Explain audit log files, only one active at a time that they’re written etc
```

Reply:
Just got rid as I'd already fleshed out those points.